common/main/en_CA.xml

```diff
@@ -1948,7 +1948,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 							<dayPeriod type="midnight">mid</dayPeriod>
 							<dayPeriod type="am">a.m.</dayPeriod>
 							<dayPeriod type="noon">↑↑↑</dayPeriod>
-							<dayPeriod type="pm">pm</dayPeriod>
+							<dayPeriod type="pm">p.m.</dayPeriod>
```

Comment:
Considering the narrow format is without periods, shouldn't we follow that here and change a.m. to am instead?

Reply:
Sure, I'll update it to be consistent that way.